tests/misc/safearrayconverttest.cpp

```diff
@@ -12,7 +12,7 @@
     #pragma hdrstop
 #endif
 
-#ifdef __WINDOWS__
+#if defined(__WINDOWS__) && !defined(__WXQT__)
```

Comment:
Hmm, I wonder if wxQt shouldn't include the OLE stuff too? If this test compiles with wxGTK, it means that even it includes it under Windows, so why not Qt too?